### PR TITLE
Update styles dependencies

### PR DESCRIPTION
**Pull request by fiskus**
* highlight.js: 10.7.2 → 11.0.1
* katex: 0.13.5 → 0.13.11
* react-ace: 9.4.0 → 9.4.1
* remarkable: 1.7.4 → 2.0.1
* sanitize.css: 11.0.1 → 12.0.1
* vega-embed: 6.17.0 → 6.18.2

### CODE REVIEW

File: catalog/app/components/Markdown/Markdown.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ import PT from 'prop-types'
 import * as R from 'ramda'
 import * as React from 'react'
 import * as RC from 'recompose'
-import Remarkable from 'remarkable'
-import { replaceEntities, escapeHtml, unescapeMd } from 'remarkable/lib/common/utils'
+import * as Remarkable from 'remarkable'
+import { linkify } from 'remarkable/linkify'
```
Comment (fiskus) [inline catalog/app/components/Markdown/Markdown.js:11]: https://github.com/jonschlinkert/remarkable/pull/351

**Review by nl0 — APPROVED** [review]
have you thoroughly tested markdown and santitize-related stuff didnt break?

**Comment by fiskus** [comment]
I opened dozen of Markdown files, and compared them to the same URLs on staging: the look the same

**Comment by fiskus** [comment]
If we don't have reference markdown file, I'll create one

**Comment by fiskus** [comment]
I noticed one change only, related to highlight.js: punctuation symbols become faded

**Comment by fiskus** [comment]
Before
Screenshot from 2021-07-08 14-50-10

After
Screenshot from 2021-07-08 14-50-17

**Comment by nl0** [comment]
> If we don't have reference markdown file, I'll create one

you can even add different code samples there so we can test both MD rendering and code highlighting via single sample file

**Comment by nl0** [comment]
> I noticed one change only, related to highlight.js: punctuation symbols become faded

that's fine, even nice

**Comment by nl0** [comment]
@fiskus did you follow highlight.js migration guide? https://github.com/highlightjs/highlight.js/blob/main/VERSION_11_UPGRADE.md

**Comment by fiskus** [comment]
Yes, our code isn't affected by those changes

**Comment by fiskus** [comment]
I want to merge after GraphQL shipping

**Pull request merged**